generator/src/main/java/com/faforever/neroxis/generator/cli/StyleOptions.java

```diff
@@ -5,14 +5,12 @@
 import picocli.CommandLine;
 
 @Getter
-public class TuningOptions {
-    @CommandLine.ArgGroup(order = 2, heading = "Options to set the generated map visibility%n")
-    private VisibilityOptions visibilityOptions;
-    @CommandLine.ArgGroup(order = 1, heading = "Options to manually tune map generation%n", exclusive = false)
-    private ParameterOptions parameterOptions;
+public class StyleOptions {
+    @CommandLine.ArgGroup(order = 2, heading = "Options to create a custom map style%n", exclusive = false)
+    private CustomStyleOptions customStyleOptions;
     private MapStyle mapStyle;
 
-    @CommandLine.Option(names = "--style", order = 1, description = "Style for the generated map. Values: ${COMPLETION-CANDIDATES}")
+    @CommandLine.Option(names = "--style", order = 6, description = "Style for the generated map. Values: ${COMPLETION-CANDIDATES}")
```

Comment:
style should still have order 1 so it is at the top

Reply:
I set it to 1, but it is still not at the top

Reply:
is it at least at the top of the group?

Reply:
This is the current output
```
Usage: generate [-hV] [--debug] [--visualize] [--map-name=<mapName>]
                [--num-to-generate=<numToGenerate>] [--out-path=<outputPath>]
                [--preview-path=<previewFolder>] [[--seed=<seed>]
                [--spawn-count=<spawnCount>] [--num-teams=<numTeams>]
                [--map-size=<mapSize>]] [[--visibility=<visibility>] |
                [--terrain-symmetry=<terrainSymmetry> | [--style=<mapStyle> |
                [[--land-density=<landDensity>]
                [--plateau-density=<plateauDensity>]
                [--mountain-density=<mountainDensity>]
                [--ramp-density=<rampDensity>]
                [--reclaim-density=<reclaimDensity>]
                [--mex-density=<mexDensity>] [--biome=<biomeName>]]]]] [COMMAND]
Generates a map from scratch
  -h, --help                 Show this help message and exit.
  -V, --version              Print version information and exit.
      --map-name=<mapName>   Name of map to recreate. Must be of the form
                               neroxis_map_generator_version_seed_options, if
                               present other parameter options will be ignored
      --out-path, --folder-path=<outputPath>
                             Folder to save the map to
      --num-to-generate=<numToGenerate>
                             Number of maps to create
      --seed=<seed>          Seed for the generated map
      --map-size=<mapSize>   Generated map size, can be specified in oGrids (e.
                               g 512) or km (e.g 10km)
                               Default: 512
      --spawn-count=<spawnCount>
                             Spawn count for the generated map
                               Default: 6
      --num-teams=<numTeams> Number of teams for the generated map (0 is no
                               teams asymmetric)
                               Default: 2
      --preview-path=<previewFolder>
                             Folder to save the map previews to
      --debug                Enable debugging
      --visualize            Enable visualization
Options to create a custom map style
      --land-density=<landDensity>
                             Land density for the generated map. Min: 0 Max: 1
      --plateau-density=<plateauDensity>
                             Plateau density for the generated map. Min: 0 Max:
                               1
      --mountain-density=<mountainDensity>
                             Mountain density for the generated map. Min: 0
                               Max: 1
      --ramp-density=<rampDensity>
                             Ramp density for the generated map. Min: 0 Max: 1
      --reclaim-density=<reclaimDensity>
                             Reclaim density for the generated map. Min: 0 Max:
                               1
      --mex-density=<mexDensity>
                             Mex density for the generated map. Min: 0 Max: 1
      --biome=<biomeName>    Texture biome for the generated map. Values:
                               BRIMSTONE, DESERT, EARLYAUTUMN, FRITHEN, MARS,
                               MOONLIGHT, PRAYER, STONES, SUNSET, SYRTIS,
                               WINDINGRIVER, WONDER
Options to specify the map style
      --style=<mapStyle>     Style for the generated map. Values: BASIC,
                               BIG_ISLANDS, CENTER_LAKE, DROP_PLATEAU, FLOODED,
                               HIGH_RECLAIM, LAND_BRIDGE, LITTLE_MOUNTAIN,
                               LOW_MEX, MOUNTAIN_RANGE, ONE_ISLAND,
                               SMALL_ISLANDS, VALLEY
Options to set the generated map visibility
      --visibility=<visibility>
                             Visibility for the generated map. Values:
                               TOURNAMENT, BLIND, UNEXPLORED
Options to set the generated map symmetry
      --terrain-symmetry=<terrainSymmetry>
                             Base terrain symmetry for the generated map.
                               Values: POINT2, POINT3, POINT4, POINT5, POINT6,
                               POINT7, POINT8, POINT9, POINT10, POINT11,
                               POINT12, POINT13, POINT14, POINT15, POINT16, XZ,
                               ZX, X, Z, QUAD, DIAG, NONE
Commands:
  biomes, --biomes  Prints the biomes available
  styles, --styles  Prints the styles available

Reply:
I'm not even sure if it makes sense to have a group heading for the last three groups, as there is only one parameter in each group.